src/styles/header.scss

```diff
@@ -0,0 +1,85 @@
+// #region header
+.header {
+  display: flex;
+  box-sizing: border-box;
```

Comment:
You don't need to specify `box-sizing: border-box;` here if you have already specified it in a global style sheet. It's a good practice to specify this property once in your global styles using the `*` selector. If you haven't done this yet, I recommend you to do it.